go/vt/vtgate/tx_conn.go

```diff
@@ -62,6 +59,16 @@ var txAccessModeToEOTxAccessMode = map[sqlparser.TxAccessMode]querypb.ExecuteOpt
 	sqlparser.ReadOnly:               querypb.ExecuteOptions_READ_ONLY,
 }
 
+type commitPhase int
+
+const (
+	COMMIT2PC_CREATETRANSACTION = iota
```

Comment:
this looks weird. do you really want iota on the three first values?
and, the naming is not camel case which I believe is the golang standard